src/scss/variables/grid.scss

```diff
@@ -1,3 +1,29 @@
+@use 'breakpoints.scss';
+
 :root {
+  --kd-grid-container-max-width: 100%;
   --kd-grid-gap: 32px;
+  --kd-grid-gap-half: 16px;
+  --kd-grid-margin: 0;
+  --kd-grid-max-width: calc(var(--kd-grid-container-max-width) - var(--kd-grid-margin) - var(--kd-grid-margin));
+
+  @media (min-width: breakpoints.$bp-md) {
+    --kd-grid-margin: 16px;
+    --kd-grid-container-max-width: #{breakpoints.$bp-md};
```

Comment:
We probably need to ask Suresh about the md breakpoint as it approaches lg. There is a large amount of whitespace and I think it would look better if the max-width went to 100% at md

 
![image](https://github.com/kyndryl-design-system/shidoka-foundation/assets/1440548/75948a77-28a9-4101-9db2-756492769258)

Reply:
I agree we need clarity on what it should be. The rule for md is 640px (fluid). It can't be both so I set it to 640px.